services/BBB.go

```diff
@@ -125,7 +125,15 @@ func (bbb BBBService) GetCycleTimeSeconds() int {
 	return bbb.Config.CycleTimeSeconds
 }
 
-func (bbb BBBService) ShouldScale(server s.Server) (s.ScaleResource, error) {
+func (bbb BBBService) ShouldScale(object s.ScaledObject) (s.ScaleResource, error) {
```

Comment:
The name is a bit misleading. I understand the name that is a function which checks, if the scaled object should scale and return a boolean. If I read the code correctly, it applies also the scaling.

Reply:
This returns an object that describes how the scaled object should be scaled and this actually gets applied later in scaler.go, maybe calling it something like ComputeScalingProposal would be more clear?

Reply:
Yes